tests/integration/pilot/vm/README.md

```diff
@@ -0,0 +1,88 @@
+# Writing Tests for VMs
+
+Support for including VMs in the integration tests has been added to the framework.
+This document describes the integration and extension mechanisms to exercise VM related code.
+The primary goals are to test VM-related code & ensure it works on a range of supported OS types and versions.
+
+**Note: We currently use mock/simulated VMs for testing purposes. In the future, the testing might switch
+to utilize actual compute instances from different providers.**
+
+## Overview
+
+This section describes the scenarios in which one might want to add a VM test:
+1. Testing existing core Istio features such as traffic management, security, telemetry, etc. for **VMs**
+2. Supporting new OS images for VMs
+3. Testing onboarding tools (iptables, certs, istio-sidecar, etc.) and workflows (services, DNS, etc.) to enmesh a VM
+
+## Secenario 1: Testing VM-related Istio Code
+
+We normally test Istio using Echo application. To test VMs' connectivity, security and telemetry,
```

Comment:
This line is a little difficult to read. Maybe:

Most integration tests in Istio use the Echo application. To test connectivity, security and telemetry for a VM in the mesh, we deploy an instance of the Echo application as a VM resource.